app/graphql/types/group_order_input_type.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Types
+  class GroupOrderFieldInputType < BaseEnum # rubocop:disable Style/Documentation
+    graphql_name 'GroupOrderField'
+    description 'Field to sort the samples by'
```

Comment:
**nitpick (blocking):** This description should use the word `groups` not `samples`

Reply:
Fixed in https://github.com/phac-nml/irida-next/pull/749/commits/8ae3cfc0e923ffd7c8defb529798b4391a4b8835